In-house patch to add support for Solaris 11.3 and 12.0.  This patch
has not yet been submitted upstream due to requirements for 3rd party
CI testing.

puppetlabs/vcsrepo is not appropriate for upstream. It removes a warning from
puppet module list.

--- a/metadata.json  2017-05-12 12:42:52.368426473 -0700
+++ b/metadata.json  2017-05-12 12:45:49.309273579 -0700
@@ -29,6 +29,14 @@
       "operatingsystemrelease": [
         "14.04"
       ]
+    },
+    {
+      "operatingsystem": "Solaris",
+      "operatingsystemrelease": [
+        "12.0",
+        "11.4",
+        "11.3"
+      ]
     }
   ],
   "description": "Installs and configures OpenStack Ironic (Bare metal).",
@@ -36,7 +44,6 @@
     {"name":"puppetlabs/inifile","version_requirement":">=1.4.0 <2.0.0"},
     {"name":"openstack/keystone","version_requirement":">=8.2.0 <9.0.0"},
     {"name":"puppetlabs/stdlib","version_requirement":">=4.9.0 <5.0.0"},
-    {"name":"openstack/openstacklib","version_requirement":">=8.2.0 <9.0.0"},
-    {"name":"puppetlabs/vcsrepo","version_requirement":">=1.3.0 <2.0.0"}
+    {"name":"openstack/openstacklib","version_requirement":">=8.2.0 <9.0.0"}
   ]
 }
